src/ApiConsole.js

```diff
@@ -811,6 +812,7 @@ export class ApiConsole extends AmfHelperMixin(LitElement) {
       .scrollTarget="${scrollTarget}"
       .serverValue="${serverValue}"
       .serverType="${serverType}"
+      .rearrangeEndpoints="${rearrangeEndpoints}"
```

Comment:
Please, pass boolean values with `?` prefix instead of `.`. This may have different results when passing incorrect, null, or undefined values.